docs/src/modules/components/MarkdownElement.js

```diff
@@ -570,159 +567,157 @@ const Root = styled('div')(
         marginBottom: theme.spacing(1),
       },
     },
-  }),
-  ({ theme }) => ({
-    [`:where(${theme.vars ? '[data-mui-color-scheme="dark"]' : '.mode-dark'}) &`]: {
```

Comment:
I think there's a reason I did not use `theme.applyDarkStyles` and go with plain string here so that it's context independent. The MarkdownElement has to use the branding theme tokens to be consistent but in some demos the branding theme is overridden by the default theme.